src/movinginterquartilemean.h

```diff
@@ -0,0 +1,44 @@
+#ifndef MOVINGINTERQUARTILEMEAN_H
+#define MOVINGINTERQUARTILEMEAN_H
+
+#include <QLinkedList>
+#include <QQueue>
+
+//! Truncated Interquartile mean
+
+//! TruncatedIQM keeps an ordered list with the last n (_capacity_)
+//! input doubles and calculates the mean discarding the lowest 25%
+//! and the highest 25% values in order to reduce sensitivity to outliers.
+//!
+//! http://en.wikipedia.org/wiki/Interquartile_mean
+class MovingInterquartileMean {
```

Comment:
It would be really nice if this (and tapfilter, but I realize the code was already this way) was in src/util. We can move it later though.